frontend/ui/elements/screen_fullscreen_menu_table.lua

```diff
@@ -0,0 +1,27 @@
+local _, android = pcall(require, "android")
```

Comment:
My apologies (as I suggested this), but if you do it this way luacheck will complain.

Maybe a little safety check like this?
```lua
local isAndroid, android = pcall(require, "android")
if not isAndroid then return end
```